### PR TITLE
display only containers created by swarm in mesos

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

File: cluster/mesos/cluster_test.go [hunk 1]
```diff
@@ -66,11 +66,6 @@ func TestContainerLookup(t *testing.T) {
...
 	assert.NotNil(t, c.Container("test-engine/container1-name1"))
 	assert.NotNil(t, c.Container("test-engine/container1-name2"))
-	// Swarm ID lookup.
-	assert.NotNil(t, c.Container("swarm1-id"))
-	// Swarm ID prefix lookup.
-	assert.NotNil(t, c.Container("swarm1-"))
-	assert.Nil(t, c.Container("swarm"))
 	// Match name before ID prefix
 	cc := c.Container("con")
 	assert.NotNil(t, cc)
```
Comment (chanwit) [inline cluster/mesos/cluster_test.go]: Not sure if testing of `com.docker.swarm.mesos.name` is also required?
I saw only a test listing `com.docker.swarm.mesos.task` here.

**Comment by chanwit** [comment]
`janky` has failed here but this PR passed on my local `aufs` box.

File: cluster/mesos/cluster_test.go (hunk 1 above)
Comment (vieux) [inline cluster/mesos/cluster_test.go]: We don't really test `com.docker.swarm.mesos.task` currently. I just pushed an update to the test.
We can leave out `com.docker.swarm.mesos.name` as it's not a mandatory field.

**Comment by chanwit** [comment]
LGTM

File: cluster/mesos/cluster_test.go (hunk 1 above)
Comment (chanwit) [inline cluster/mesos/cluster_test.go]: I see.

**Comment by vieux** [comment]
ping @aluzzardi @abronan

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**